server/container_create_linux.go

```diff
@@ -715,6 +716,19 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		return nil, err
 	}
 
+	if containerIDMappings == nil {
+		rootPair = idtools.IDPair{UID: 0, GID: 0}
```

Comment:
should this be owned by the UID/GID in the container?

Reply:
`ctr.Spec().Config.Process.User.UID`?
The pr originally set it to the container, but failed all the ci tests since the code creating the /etc. So I use the default root id in the container if the containerIDMappings == ni.